### ISSUE TITLE
Highlights for PaperColor hard to read

### ISSUE THREAD

**Issue by MaxG87** [ggandor/leap.nvim#63]
Thank you very much for creating Leap. I am very fascinated by the concept and would love to adopt it.

I use Vim and NeoVim with the [PaperColor](https://github.com/NLKNguyen/papercolor-theme) colour scheme. When I gave Leap a try, I noticed that the colour of the hint boxes makes it very hard to recognise the contained letter. I attached a screenshot for demonstration.

![grafik](https://user-images.githubusercontent.com/5477952/195311138-de507f70-49fc-4215-8fd5-b1db95774798.png)

I wasn't able to figure out how to adjust the colours on my own, given the documentation. Would it be possible to extend the documentation a bit, maybe by giving an example?

Of course, I would highly appreciate if Leap's colours could be made compatible with PaperColor.

**Comment by ggandor** [comment]
See `:help nvim_set_hl`.

Example:
```
-- add "lua" before the line if you're using Vimscript
`vim.api.nvim_set_hl('0', 'LeapLabelPrimary', { fg = 'white', bg = #eeaaaa, bold = true }
```

> the colour of the hint boxes makes it very hard to recognise the contained letter

Interesting. The default (light) background for labels is much lighter. I just tried Leap with PaperColor, and looks fine (different than in your screenshot). 

> I would highly appreciate if Leap's colours could be made compatible with PaperColor.

The simplest way is to open an issue in the PaperColor repo then. Leap provides default light and dark colors, but it's impossible to make them compatible with every existing colorscheme.

**Comment by MaxG87** [comment]
Thank you for your quick reply. It helped me to sort out the issue.

Besides using PaperColor for Vim, I also use [PaperColour for Konsole](https://github.com/MaxG87/konsole-papercolor). Apparently this influences all cterm colours. In particular, `Color1` was used as background colour for the unconfigured Leap. Hopefully this helps other users of Konsole.

Figuring out  how to use `nvim_set_hl` was quite a bumpy ride. First, how to use `nvim_set_hl` differs, depending on whether it appears in the `.vimrc` or is used from inside a running NeoVim. Second, at least in my case, the arguments `fg` and `bg` where useless, I had to resort to `ctermfg` and `ctermbg` instead.

What I put into my `.vimrc` was:

    lua vim.api.nvim_set_hl(0, 'LeapLabelPrimary', {ctermfg="White", ctermbg="DarkBlue", bold = true })

The version of this command that can be used in command mode:

    call nvim_set_hl(0, 'LeapLabelPrimary', { 'ctermfg': 'White', 'ctermbg': 'DarkBlue', 'bold': v:true })

I hope this helps other users of PaperColor und Leap who end up here.

**Comment by desbo** [comment]
@MaxG87 thanks! I had the same problem although I'm using Nord rather than PaperColour, and leap's highlights were red before adding your command.

**Comment by MaxG87** [comment]
Indeed, thanks to your comment I discovered a flaw in my configuration that prevented that I set `termguicolors`. Now the `fg` and `bg` arguments for `nvim_set_hl` work and I do not necessarily need the above fix.
